runtime_scan/pkg/orchestrator/scanconfigwatcher/schedule.go

```diff
@@ -241,14 +281,19 @@ func NewScanConfigSchedule(scanConfig *models.ScanConfig, window *ScheduleWindow
 		}, nil
 	}
 
+	// Check whether the operationTime is before the window meaning that it is in the past compared to the window.
 	if window.Before(operationTime.Time()) {
```

Comment:
As mentioned I think if we fix these functions to align with the time package it'll read much better this will become:

if window.After(operationTime.Time()) {
}

which is much better "pseudo english": "if the schedule window is after the operation time; then ... "

Reply:
Fixed.